src/components/Petitions/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PetitionListItem from 'components/PetitionListItem';
+import Pager from 'components/Pager';
+
+const Petitions = ({ total, petitions, page, per }) => (
+  <div>
+    <ul>
+      {petitions.map((petition, index) => {
+        return (
+          <PetitionListItem {...petition} key={petition.id} />
+        );
+      })}
```

Comment:
here you can simplify to

`{petitions.map((petition, index) => <PetitionListItem {...petition} key={petition.id} />)}`